### ISSUE TITLE
Update from deepCoadd_calexp_calib to deepCoadd_calexp_photoCalib

### ISSUE THREAD

**Issue by heather999** [LSSTDESC/DC2-analysis#85]
As mentioned here: https://github.com/LSSTDESC/DC2-production/issues/344
This also affects https://github.com/LSSTDESC/DC2-analysis/blob/c8f2c16e25a45ad9b0c7761401c0d08aed774bc3/tutorials/dm_butler_lensing_cuts.ipynb#L98
Without this change, tests running on this notebook fail with the new version of the desc-stack jupyter kernel using `w_2019_19`
Michael had mentioned a desire to maintain backward compatibility.  Is that necessary?  It's not the data, but rather the version of the DMstack used that dictates the template used, so as long as we're all running with >= Run1.2 and using DMstack ~ `w_2019_19`, this modification would be ok, I think.

**Comment by heather999** [comment]
Moving to `deepCoadd_calexp_photoCalib` produces some warnings: 
```
/opt/lsst/software/stack/python/miniconda3-4.5.12/envs/lsst-scipipe-1172c30/lib/python3.7/site-packages/ipykernel/__main__.py:45: FutureWarning: Call to deprecated function (or staticmethod) setThrowOnNegativeFlux. (No-op: PhotoCalib never throws on negative instFlux (will be removed after v18).)
/opt/lsst/software/stack/python/miniconda3-4.5.12/envs/lsst-scipipe-1172c30/lib/python3.7/site-packages/ipykernel/__main__.py:58: FutureWarning: Call to deprecated method getMagnitude. (For backwards compatibility with Calib; use `instFluxToMagnitude` instead (will be removed after v18).)
/opt/lsst/software/stack/python/miniconda3-4.5.12/envs/lsst-scipipe-1172c30/lib/python3.7/site-packages/ipykernel/__main__.py:59: FutureWarning: Call to deprecated method getMagnitude. (For backwards compatibility with Calib; use `instFluxToMagnitude` instead (will be removed after v18).)
```

Planning to remove: 
`calib.setThrowOnNegativeFlux(False)`

and to modify

``` 
data[filter_ + '_mag_CModel'] = calib.getMagnitude(forced['modelfit_CModel_instFlux'])
data[filter_ + '_mag_err_CModel'] = calib.getMagnitude(forced['modelfit_CModel_instFluxErr'])
```

**Comment by yymao** [comment]
Fixed by #89
